interfaces/builtin/content_test.go

```diff
@@ -195,11 +199,11 @@ apps:
 
 func (s *ContentSuite) TestResolveSpecialVariable(c *C) {
 	info := snaptest.MockInfo(c, "name: name", &snap.SideInfo{Revision: snap.R(42)})
-	c.Check(builtin.ResolveSpecialVariable("foo", info), Equals, "/snap/name/42/foo")
```

Comment:
Silly question - if the code got changed to use /snap as static prefix, do we still need to change:
```
- "/snap/foo"
+ filepath.Join(dirs.CoreSnapMountDir, "foo")
```
I guess the answer is yes to make it more obvious when we mean "context-is-inside-core" and "context-is-outside"(?)